spec/ics-027-interchain-account/README.md

```diff
@@ -0,0 +1,283 @@
+---
+ics: 27
+title: Interchain Account
+stage: Draft
+category: IBC/TAO
+author: Tony Yun <yunjh1994@everett.zone>, Dogemos <dogemos@dogemos.com>
+created: 2019-08-01
+modified: 2019-09-24
+---
+
+## **Synopsis**
+
+This standard document specifies packet data structure, state machine handling logic, and encoding details for the account management system over an IBC channel between separate chains.
+
+### **Motivation**
+
+On Ethereum, there are two types of accounts: externally owned accounts, controlled by private keys, and contract accounts, controlled by their contract code [[ref](https://github.com/ethereum/wiki/wiki/White-Paper)]. Similar to Ethereum's CA(contract accounts), Interchain accounts are managed by another chain(zone) while retaining all the capabilities of a normal account (i.e. stake, send, vote, etc). While an Ethereum CA's contract logic is performed within Ethereum's EVM, Interchain accounts are managed by another chain via IBC in a trustless way.
+
+### **Definitions**
+
+The IBC handler interface & IBC relayer module interface are as defined in [ICS 25](https://github.com/cosmos/ics/blob/bez/15-ics-cosmos-signed-messages/spec/ics-025-handler-interface) and [ICS 26](https://github.com/cosmos/ics/blob/bez/15-ics-cosmos-signed-messages/spec/ics-026-relayer-module), respectively.
+
+### **Desired Properties**
+
+- Permissionless
+- Fault containment: Interchain account must follow rules of its dwelling chain, even in times of byzantine behavior by the counterparty chain (the chain that manages the account)
+- The chain(that controls the account) requesting a specific transaction from must process the results asynchronously and according to the chain's logic.
+- Sending and receiving transactions will be processed in an ordered manner.
+
+## **Technical Specification**
+
+### **Data Structures**
+
+`RegisterIBCAccountPacketData` is used for counterparty chain to register an account. Interchain account's address is defined deterministically with channel path and salt. Process of determining address is influenced by [EIP-1014](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1014.md). It allows interactions to be made with addresses that do not exist yet on-chain. These specificities will allow more offchain services to build on top of the data structure.
+
+```typescript
+interface RegisterIBCAccountPacketData {
+  salt: Uint8Array
+}
+```
+
+`RunTxPacketData` is used to run tx for interchain account. Tx bytes is dependent on app, and it is formed with minimal data set except data for validating signatures.
```

Comment:
You mean without the signatures, right?

Reply:
Right. I think that application logic can know what chain requests transaction and whether that chain has the right permission. So, there is no need to include a signature to the interchain account.